packages/app/src/hooks/useFetchMetricResourceAttrs.tsx

```diff
@@ -0,0 +1,120 @@
+import { ResponseJSON } from '@clickhouse/client';
+import { chSql } from '@hyperdx/common-utils/dist/clickhouse';
+import { SourceKind } from '@hyperdx/common-utils/dist/types';
+import { TSource } from '@hyperdx/common-utils/dist/types';
+import { useQuery } from '@tanstack/react-query';
+
+import { getClickhouseClient } from '@/clickhouse';
+import { formatAttributeClause } from '@/utils';
+
+const METRIC_FETCH_LIMIT = 25;
```

Comment:
just an arbitrary limit I chose. Happy to adjust as needed

Reply:
I think we should be able to safely set this into the thousands?

Reply:
Ok cool. I started low to error on the side of caution. I also saw in V1 there was a top 10 suggestion limit of some sort.